config/app.yaml

```diff
@@ -1,6 +1,6 @@
 port: 8989
 base_url: "/api/1.0"
-verify_root_dir: "data/verify/"
+verify_root_dir: "/data/verify/"
```

Comment:
Should this really default to a directory on root? I think for development purposes it might be better if it defaulted to `data/verify/` as it was before.

Reply:
Fixed.